### PR TITLE
Roadmap gridlayout corrected Fixes #48

### LINKED ISSUE #48: Replace LinearyLayout manager with GridLayoutManager in RoadmapAdapter
### Replace LinearyLayout manager with GridLayoutManager in the recycler view used in the file `RoadmapAdapter` 
- Remove staggered grid layout
- Look for a workaround for providing margin/padding to card based on the position
- Equal width of the roadmap cards in the grid (2 cols)

### Possible Solution
- Check if the `position` inside the Recycler view adapter is odd/even, based on that add padding to the right or left
 Something like this

  ```
    if(position % 2 == 0)
    {
    container.setPaddingRight(16);
    }
  ```

### PR DESCRIPTION
**Pull request by adarsh434**
## What was the issue?
 Grid Layout was not appropriate inside the roadmap recyclerview.

## What does this PR do?
- Corrects the height and width of Roadmap card.
- Replaces LinearLayoutManager with GridLayoutManager.
- Solves #48

### CODE REVIEW

File: app/src/main/java/com/iet/ietians_diary/RoadmapFragment.java [hunk 1]
```diff
@@ -92,8 +93,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         RoadmapRecyclerViewAdapter RoadmapAdapter = new RoadmapRecyclerViewAdapter(list, getContext());
         roadmap.setAdapter(RoadmapAdapter);
 
-        StaggeredGridLayoutManager staggeredGridLayoutManager = new StaggeredGridLayoutManager(2, StaggeredGridLayoutManager.VERTICAL);
-        roadmap.setLayoutManager(staggeredGridLayoutManager);
+//        StaggeredGridLayoutManager staggeredGridLayoutManager = new StaggeredGridLayoutManager(2, StaggeredGridLayoutManager.VERTICAL);
```
Comment (Anuragtech02) [inline app/src/main/java/com/iet/ietians_diary/RoadmapFragment.java:96]: Please remove this comment, no need.

**Review by Anuragtech02 — CHANGES_REQUESTED** [review]
Looks good, just one change and we'll be good to go!

File: app/src/main/java/com/iet/ietians_diary/RoadmapFragment.java (hunk 1 above)
Comment (adarsh434) [reply app/src/main/java/com/iet/ietians_diary/RoadmapFragment.java]: Thanks for the review, I have removed the comment.

**Review by Anuragtech02 — APPROVED** [review]
LGTM! Thanks ;)

**Pull request merged**